### PR TITLE
Improve website accessibility

### LINKED ISSUE #170: Make site more accessible
## Summary

There are a lot of things we can do to make the site more accessible, see the audit below:

screen shot 2018-10-29 at 10 58 29 am

### Motivation

Make the site more accessible!

### PR DESCRIPTION
**Pull request by rosslh**
## Proposed Change

Closes #170

### How did you do this?

Semantic HTML

### Why are you choosing this approach?

Improve accessibility for non-mouse users and people with disabilities

### Any open questions you want to ask code reviewers?

Nope

### List of changes:

- Improve accessibility by fixing eslint warnings
- Upgrade eslint warnings to errors

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit 6b732709b8b8578675bf178069895c0e7764f9c1

https://deploy-preview-212--qhacks-website.netlify.com

**Comment by RobertWSaunders** [comment]
Definitely getting better, want to address the outstanding items:

screen shot 2018-11-02 at 1 44 34 pm

**Comment by RobertWSaunders** [comment]
Also, you can run the test by opening Chrome and going to the site, then opening dev tools and clicking "Audit" at the top. FYI there are also manual checks we can perform to make it even more accessible:

screen shot 2018-11-02 at 1 52 50 pm

**Comment by RobertWSaunders** [comment]
Also a few best practices have been broken, lets update those as well so we can get back to 100 there:
screen shot 2018-11-02 at 1 53 56 pm

File: src/components/MobileMenu.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ const MobileMenu = (props) => (
       }
     }}
   >
-    <img
+    <input
```
Comment (joeytepp) [inline src/components/MobileMenu.js]: This change seems to have created a text field on the menu, not sure how that can be prevented

**Comment by rosslh** [comment]
@RobertWSaunders @joeytepp made changes based on your feedback! Lighthouse accessibility and best practices scores brought up to :100:!

File: src/components/BriefSummary.js [hunk 2]
```diff
@@ -83,6 +83,7 @@ const BriefSummary = () => (
           }}
```
Comment (joeytepp) [inline src/components/BriefSummary.js]: Could we change the apostrophe above this to a `&#39;` for consistency and to fix the diff?

**Pull request merged**